pkg/protocol/request.go

```diff
@@ -450,6 +450,20 @@ func (req *Request) SetQueryString(queryString string) {
 	req.URI().SetQueryString(queryString)
 }
 
+// SetQueryParams sets query Params.
+func (req *Request) SetQueryParams(params map[string]string) {
+	var builder strings.Builder
+	builder.Write([]byte("?"))
+	for k, v := range params {
+		builder.Write([]byte(k))
```

Comment:
What happens if there are special characters? Do I need to use `url.QueryEscape`?
```go
	{
		"name": "cx&k=1",
		"age":  "25",
	}
```